### ISSUE TITLE
Intermediary firmware not offered during onboarding/updating of old device

### ISSUE THREAD

**Issue by bosomt** [trezor/trezor-suite#3911]
**Describe the bug**
Indermediary firmware was introduced to remove need to install 1.6.3 before you can install latest firmware.
But suite 21.6.2 still offers 1.6.3 

**Steps to reproduce:**
1. Connect device with firmware older than 1.6.3
2. Start onboarding or update of device
3. Observe offered version

**Info:**
 - Suite version: desktop 21.6.2 (4e29f3d6e527618670936bf6d1d3617b217be30c)
 - Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/21.6.2 Chrome/89.0.4389.69 Electron/12.0.0 Safari/537.36
 - OS: Win32
 - Screen: 1280x720
 - Device: model One 1.6.0 regular

**Screenshots:**
![obrázok](https://user-images.githubusercontent.com/31506317/122415136-f1d90b80-cf87-11eb-8f76-1e2207802dd1.png)

**Comment by tsusanka** [comment]
@bosomt so this happens also for new 1.4.0 devices in onboarding? So in another words the intermediary FW is never offered?

**Comment by bosomt** [comment]
@tsusanka 

sorry I only have two devices older that 1.6.x
1.3.1 BL 1.6.0 FW - upgrade to 1.6.3 is offered
1.5.0 BL 1.6.1 FW - upgrade to 1.10.1 is offered

see screenshot

I think that there is some error in wording.
Why does it say `Your device is already updated to latest firmware`


![obrázok](https://user-images.githubusercontent.com/31506317/122456476-e8fb3080-cfad-11eb-9bb3-ce8bdfca9511.png)

**Comment by tsusanka** [comment]
And both of these devices were <s>not</s> initialized right? Because the first thing might be connected to https://github.com/trezor/trezor-suite/issues/3727 maybe?

Edit: I actually wanted to say that I suspect them being initialized :facepalm:.

**Comment by bosomt** [comment]
@tsusanka  
1.5.0 BL 1.6.1 FW     initialized: True
1.3.1 BL 1.6.0 FW      initialized: True,

do you want me to wipe seed OR firmware and retest onboarding behavior ?
i will send separate issue for onboarding wording

**Comment by tsusanka** [comment]
Wipe seed only. To clarify: my process of thinking is that it might work correctly in onboarding but we have forgotten about this dialog which basically says "your FW is not supported anymore, please update" and we do not utilize the intermediary FW there.

**Comment by tsusanka** [comment]
But as I think of it it does not make sense, because 1.6.1 FW is not supported as well.. Well, please retry with wiped seed and let's see.

**Comment by bosomt** [comment]
So i tested empty BL 1.3.2 device

1) intermediary firmware was installed,
2) I was asked to reconnect device
3) I was offered update device from 1.10.0 to 1.10.1 
4) Confirmed,installed auto rebooted.

attaching log
[trezor-suite-log-1624006347775.txt](https://github.com/trezor/trezor-suite/files/6676158/trezor-suite-log-1624006347775.txt)


```
2021-06-18T09:37:32.137Z - DEBUG(csp): Header applied to https://data.trezor.io/firmware/1/trezor-inter-1.10.0.bin
2021-06-18T09:37:31.369Z - WARN(console-log): Cannot install latest firmware. Will install intermediary fw instead. - 

```



![image](https://user-images.githubusercontent.com/31506317/122541357-d6264180-d029-11eb-9aeb-3d49c46df4fb.png)

**Comment by tsusanka** [comment]
So that seems correct right? :)

**Comment by tsusanka** [comment]
So we could assume this concerns only initialized devices?

**Comment by bosomt** [comment]
@tsusanka yes we can

**Comment by tsusanka** [comment]
@bosomt and upon clicking "Install firmware" you indeed get offered 1.6.3 right? I mean do not to through with it because you do not want to get rid of your old device but I am just curious :). Is there a way to find out actually? :thinking: Maybe check what is being downloaded from data.trezor.io?

**Comment by bosomt** [comment]
@tsusanka firmware is installed, and 1.10.1 fw is installed too but suite installs it over and over again with this dialogue till device glitched and crashed and I ended up with device with bootloader 1.10.0 and no seed or firmware. Attached video of that loop + log from suite.

https://youtu.be/044_JQynLK4
[trezor-suite-log-1624345728858.txt](https://github.com/trezor/trezor-suite/files/6692021/trezor-suite-log-1624345728858.txt)


![image](https://user-images.githubusercontent.com/31506317/122880287-23f8bd80-d33a-11eb-833c-0912c8d527db.png)

```
**Info:**
 - Suite version: desktop 21.6.2 (1ff4cbc91f640b645aa72c59047ffeff01819b1b)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/21.6.2 Chrome/89.0.4389.69 Electron/12.0.0 Safari/537.36
 - OS: Linux x86_64
 - Screen: 2560x1440
```

**Comment by matejkriz** [comment]
the original issue is resolved and the current version of firmware should be offered for all devices, but the issue with intermediary firmware on old and already initialized devices still remains and has to be addressed.

**Comment by sorooris** [comment]
@matejkriz i apologize but i am quite confused how there are two separate issues - isn't there only one issue with the intermediary? Could you please explain what the other issue is if not?

**Comment by matejkriz** [comment]
@sorooris latest version should be always offered, but I'm not sure if https://github.com/trezor/trezor-suite/issues/3911#issuecomment-865665663 is resolved

**Comment by sorooris** [comment]
@matejkriz Okay, thanks, we'll look into it then!

I'll split the issue if https://github.com/trezor/trezor-suite/issues/3911#issuecomment-865665663 persists.

**Comment by STew790** [comment]
I just tested what @bosomt reported on staging Suite and I got the same result.

I initiated an update with a device with FW 1.6.0 (BL 1.3.2) and after installing the intermediary FW it keeps trying to update to the latest FW and gets stuck in a loop. I attach a screenshot from the console below.

![image](https://user-images.githubusercontent.com/48477147/128714767-3ee44885-38df-4c60-9828-21e4a6084127.png)
